src/Jobs/CrawlSiteJob.php

```diff
@@ -24,11 +25,18 @@ class CrawlSiteJob implements ShouldQueue
 
     public $timeout = 60 * 5;
 
+    public $uniqueFor = 60 * 60;
+
     public function __construct(
         public SiteSearchConfig $siteSearchConfig
     ) {
     }
 
+    public function uniqueId(): string
+    {
+        return $this->siteSearchConfig->index_base_name;
```

Comment:
Is there a particular reason why we would use `index_base_name` over `->getKey()`?

Reply:
No particular reason